Content.Server/Chemistry/Components/ChemMasterComponent.cs

```diff
@@ -15,6 +15,12 @@ public sealed partial class ChemMasterComponent : Component
         [DataField("pillType"), ViewVariables(VVAccess.ReadWrite)]
         public uint PillType = 0;
 
+        [DataField("pillNumber"), ViewVariables(VVAccess.ReadWrite)]
```

Comment:
```suggestion
        [DataField]
```
This is all applied automatically

Reply:
Ah okay, good to know!